cf_units/_udunits2_parser/graph.py

```diff
@@ -109,3 +109,35 @@ class Timestamp(Terminal):
     # This is likely to change in the future, but there are some
     # gnarly test cases, and should not be undertaken lightly.
     pass
+
+
+class Visitor:
+    """
+    This class may be used to help traversing an expression graph.
+
+    It follows the same pattern as the Python ``ast.NodeVisitor``.
+    Users should typically not need to override either ``visit`` or
+    ``generic_visit``, and should instead implement ``visit_<ClassName>``.
+
+    Remember that each method MUST call ``generic_visit(node)`` if a node's
+    children should be processed.
```

Comment:
Perhaps add "alternatively, each of the children can be visited individually using the visit method (e.g. ``visit(node.lhs)``)"?